src/client/gui/assets/multipass.gui.autostart.desktop

```diff
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Name=Multipass
-Exec=multipass.gui --autostarting
+Exec=multipass.gui
```

Comment:
Do we need a separate file then?

Reply:
You mean separate from the `data/multipass.gui.desktop` one? The autostart desktop file has to reside in the Flutter App's assets directory, so that it gets bundled with the app and can be accessed by the app. But I just tested to see if a symlink works. And it seems to work, since when the app is built, it copies the actual file to the app asset bundle, which is what we need. I'll change it.

Reply:
Yeah, I was thinking something like that (assuming that the regular desktop file has all that we need).

Reply:
Well, apparently the packaged snap does not copy the file from the symlink into the assets bundle, and I don't really understand why. I'll just go to the regular file for the time being. I think the symlink might be a very micro optimization, given that it's a very small file that is very unlikely to change.